tsconfig.json

```diff
@@ -12,10 +12,6 @@
   "exclude": [
     "built",
     "node_modules",
-    "testapp",
-    "typings/browser",
-    "typings/browser.d.ts",
-    "typings/main",
-    "website"
+    "typings/globals"
```

Comment:
You need to add back in 'testapp' and 'website'. If not, tsc will try to do transpile the website and the testapp directory